WORKSPACE

```diff
@@ -8,3 +8,9 @@ cc_configure()
 
 load("@envoy_api//bazel:repositories.bzl", "api_dependencies")
 api_dependencies()
+
+load("@io_bazel_rules_go//go:def.bzl", "go_rules_dependencies", "go_register_toolchains")
```

Comment:
Doesn't this need to get added to CI workspaces also? And does this mean that all consumers also have to modify their workspace also?

Reply:
Removed. Not needed unless trying to build go rules.